doc-examples/jdbc-spring-template-example-groovy/src/main/groovy/example/AbstractBookRepository.groovy

```diff
@@ -18,15 +18,15 @@ import javax.sql.DataSource
 @JdbcRepository(dialect = Dialect.H2)
 abstract class AbstractBookRepository implements CrudRepository<@Valid Book, @NotNull Long> {
 
-    private final JdbcTemplate jdbcTemplate
+    private final JdbcTemplate jdbcTemplate; //<2>
```

Comment:
I know this is from merge, but `//<2>` is repeated below, should be removed from here I think.